DEPRECATIONS.md

```diff
@@ -4,32 +4,32 @@ This document lists deprecated constructs in the SDK, with their expected time 
-| `GlobalState::isInitialized` | 2022.07.19<br/>5.6.0 | 2022.08.06 ⌛ | 2022.09.06 ⌛ | Use `ClientState:isInitialized` instead.|
-| `GlobalState::isConnecting` | 2022.07.19<br/>5.6.0 | 2022.08.06 ⌛ | 2022.09.06 ⌛ | Use `ClientState:isConnecting` instead.|
-| `GlobalState::isOffline` | 2022.07.19<br/>5.6.0 | 2022.08.06 ⌛ | 2022.09.06 ⌛ | Use `ClientState:isOffline` instead.|
-| `GlobalState::isOnline` | 2022.07.19<br/>5.6.0 | 2022.08.06 ⌛ | 2022.09.06 ⌛ | Use `ClientState:isOnline` instead.|
-| `GlobalState::connectionState` | 2022.07.19<br/>5.6.0 | 2022.08.06 ⌛ | 2022.09.06 ⌛ | Use `ClientState:connectionState` instead.|
-| `GlobalState::initialized` | 2022.07.19<br/>5.6.0 | 2022.08.06 ⌛ | 2022.09.06 ⌛ | Use `ClientState:initialized` instead.|
-| `GlobalState::user` | 2022.07.19<br/>5.6.0 | 2022.08.06 ⌛ | 2022.09.06 ⌛ | Use `ClientState::user` instead.|
-| `GlobalState::errorEvents` | 2022.07.19<br/>5.6.0 | 2022.08.06 ⌛ | 2022.09.06 ⌛ | This method is no longer used.|
-| `GlobalState::typingUpdates` | 2022.07.04 <br/>5.5.0 | 2022.07.19<br/>5.6.0 | 2022.08.01 ⌛ | Use `GlobalState::typingChannels` instead. |
-| `MessageListView.setUserBlockHandler` | 2022.07.04 <br/>5.5.0 | 2022.07.19<br/>5.6.0 | 2022.08.01 ⌛ | The block action has been removed. Use `MessageOptionItemsFactory.setMessageOptionItemsFactory()` in conjunction with `MessageOptionItemsFactory.setCustomActionHandler()` to add support for custom block action. |
+| `ChannelListView.showLoadingMore()` | 2022.08.02 <br/>5.7.0 | 2022.09.06 ⌛ | 2022.10.04 ⌛ | Insert the loading item before passing the list to the adapter. |
+| `ChannelListView.hideLoadingMore()` | 2022.08.02 <br/>5.7.0 | 2022.09.06 ⌛ | 2022.10.04 ⌛ | Insert the loading item before passing the list to the adapter. |
+| `RowScope.DefaultComposerInputContent` | 2022.08.02<br/>5.7.0 | 2022.09.06 ⌛ | 2022.10.04 ⌛ | Use `MessageInput` instead. |
+| `LegacyDateFormatter` | 2022.08.02<br/>5.7.0 | 2022.08.16 ⌛ | 2022.08.30 ⌛ | The class is unused and will be removed. |
+| `PorterImageView` | 2022.08.02<br/>5.7.0 | 2022.08.16 ⌛ | 2022.08.30 ⌛ | The class is unused and will be removed. |
+| `PorterShapeImageView` | 2022.08.02<br/>5.7.0 | 2022.08.16 ⌛ | 2022.08.30 ⌛ | The class is unused and will be removed. |
+| `ChatClient::disconnect` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.19 ⌛ | Use `ChatClient.disconnect(Boolean)` instead. |
+| `ChatLogger` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.13 ⌛ | Use `StreamLog` instead.|
+| `ChatLogger.Config` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.13 ⌛ | Use `ChatLoggerConfig` instead.|
+| `ChatLogger::get` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.13 ⌛ | Use `StreamLog::getLogger` instead.|
+| `BaseChatModule::logger` | 2022.07.19<br/>5.6.0 | 2022.08.16 ⌛ | 2022.09.13 ⌛ | Use `StreamLog` instead.|
+| `GlobalState::setUser` | 2022.07.19<br/>5.6.0 | 2022.08.02<br/>5.7.0 | 2022.09.06 ⌛ | Use `ClientMutableState:setUser` instead.|
```

Comment:
There is no `GlobalState::setUser` field. Previously we had MutableGlobalState::setUser but it has already been removed and the class itself is internal. I would just remove this line completely.

Reply:
nice catch. Thank you!